### ISSUE TITLE
Text not displaying properly on mobile devices

### ISSUE THREAD

**Issue by sani-rus** [GuilhermeFerreiraAraujo/SimencPlastika#2]
I think that the problem of not showing all the text on mobile devices is caused by the following facts:
- text is in white colour
- background image is grey. On the desktop background image covers whole area of a page, and the white text is displayed correctly on the grey background. 
- on the mobile devices background image does not cover the whole page, so bottom part of the page is white background, and white text is illegible. 

So in summary text is displayed "correctly", but as white on white.

Can you set an option that background image would cover whole page also on the mobile devices?

Additional information: setting the background-repeat to repeat in mainDiv didn't solve the problem.

**Comment by GuilhermeFerreiraAraujo** [comment]
The image was supposed to not move, the problem is that the property "background-attachment: fixed" was disabled for mobiles. I just change the background color and remove the image for mobile devices, it should fix the problem.
